nixos/modules/services/hardware/triggerhappy.nix

```diff
@@ -102,7 +102,6 @@ in
 
     systemd.services.triggerhappy = {
       wantedBy = [ "multi-user.target" ];
-      after = [ "local-fs.target" ];
```

Comment:
Why remove the `after` clause? Won't this change the semantics?

Reply:
No - as described in the PR description:

Since #61321, local-fs.target is
part of sysinit.target again, meaning units without
`DefaultDependencies=no` will automatically depend on it, and the manually set dependencies can be dropped.

`triggerhappy.service` doesn't set `DefaultDependencies=no` (ususally only some low-level, mostly systemd-internal services do), so it depends on `sysinit.target`, which pulls in `local-fs.target`.

Reply:
But IUC `After=` is not about setting dependencies, it only regulates ordering whenever units are started/stopped concurrently. From the manual:

> Note that this setting is independent of and orthogonal to the requirement dependencies as configured by Requires=, Wants= or BindsTo=